src/stop.sh

```diff
@@ -42,6 +43,10 @@ do_killall() {
     $SUDO killall -u $MYNAME $1
 }
 
+do_killcephadm() {
+    $SUDO $CEPHADM_DIR_PATH/vstart-cleanup.sh
```

Comment:
I don't think we need that. 

`cephadm rm-cluster --fsid $fsid --force`

should be simpler. 

Note that vstart supports restarting existing cluster, which is not what we're doing here. A similar operation would be something like `cephadm ls | jq .[].systemd_unit | xargs -n 1 systemctl disbale